docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -1 +1,97 @@
-# Shaka Packager
\ No newline at end of file
+# Shaka Packager
+
+## Setup using Docker
+
+- First download and install Docker - https://www.docker.com/get-started
+
+- Pull the latest Shaka Packager Docker image by entering ```docker pull google/shaka-packager``` in your terminal.
+
+- Now run the Docker image by entering ```docker run -v /<file_path_to_content>/:/media -it --rm google/shaka-packager``` in your terminal. This will replicate your directory and create a ```media``` file path within the Docker container.
+
+- Test that setup is correct, by analyising a file within the filepath. You can do this by entering the command ```packager input=/media/<name_of_content>.mp4 --dump_stream_info```.
+
+If successful you should then get a dump of the stream information which should look something like this:
```

Comment:
These steps can be written in a simplified way. Why not have a code block (a script) with comments:
```
# pull and run shaka packager 
docker pull google/shaka-packager
docker run -v /<file_path_to_content>/:/media -it --rm google/shaka-packager

# Test that setup is correct by getting stream info
packager input=/media/<name_of_content>.mp4 --dump_stream_info
```

Expected output...

Reply:
Looks better for sure. Actioned.